sample.conf

```diff
@@ -118,40 +142,45 @@ PageType	cgi
 
 #DNSChildren	0
 
-# HTMLPre defines HTML code to insert at the very beginning of the
-# file. Use it for server-side script code, like PHP.
+# HTMLPre allows code to be inserted at the very beginning of the HTML files.
+# Be careful not to include any HTML here, as it is inserted before the
+# <html> tag in the file. Use it for server-side scripting capabilities,
+# such as PHP, to insert scripting files and other directives.
 
 #HTMLPre 
 
-# HTMLHead defines HTML code to insert within the <HEAD></HEAD>
-# block, immediately after the <TITLE> line.  Maximum line length
-# is 80 characters, so use multiple lines if needed.
+# HTMLHead defines HTML code to insert within the <HEAD></HEAD> block.
+# There is no default. Useful for adding scripts to the HTML page, such as
+# Javascript or php3, or even just for adding a few META tags to the document.
 
 #HTMLHead <META NAME="author" CONTENT="The Webalizer">
 
-# HTMLBody defined the HTML code to be inserted right after the opening
-# <body> tag.
+# HTMLBody defined the HTML code to be inserted right after the start tag of
+# the report, just before the title and "summary period/generated on" lines.
+# Keep in mind the placement of this code in relation to the title and other
+# aspects of the web page. A typical use is to add a corporate logo (graphic)
+# in the top right.
+# It is an error to have the <body> tag in any HTMLBody entries.
 
 #HTMLBody 
 
-# HTMLPost defines the HTML code to insert immediately before the
-# first <HR> on the document, which is just after the title and
-# "summary period"-"Generated on:" lines.  If anything, this should
-# be used to clean up in case an image was inserted with HTMLBody.
-# As with HTMLHead, you can define as many of these as you want and
-# they will be inserted in the output stream in order of apperance.
-# Max string size is 80 characters.  Use multiple lines if you need to.
+# HTMLPost defines HTML code that is placed after the title and "summary
+# period/generated on" lines, just before the initial horizontal rule <hr> tag.
+# Normally this keyword isn't needed, but is provided in case you included a
+# large graphic or some other weird formatting tag in the HTMLBody section that
+# needs to be cleaned up or terminated before the main report section.
 
-#HTMLPost 	
+#HTMLPost 	<BR CLEAR="all">
```

Comment:
The `clear` attribute for the `br` element is deprecated in favor of CSS.

Reply:
I'll remove this change then as well. Just thought it a "clear" example :wink: